src/main/java/org/jenkinsci/plugins/builduser/BuildUser.java

```diff
@@ -60,8 +59,11 @@ public void setUp(Context context, Run<?,?> build, FilePath workspace,
 
     /**
      * Retrieve user cause that triggered this build and populate variables accordingly
+     *
+     * @param build     The build that was triggered
+     * @param variables A map to store build user properties in
      */
-    private void makeUserBuildVariables(@Nonnull Run build, @Nonnull Map<String, String> variables) {
+    public static void makeUserBuildVariables(@Nonnull Run build, @Nonnull Map<String, String> variables) {
```

Comment:
This method is the one that actually extracts user information from a `Run` and sets it in a `Map`. It doesn't do anything that depends on state of the `BuildUser` object, so it doesn't need to be an instance method.

By making it public and static, I can use it from the pipeline step code - ensuring that both methods of obtaining Build User vars use the exact same logic, and that there is only ever one place to update in the future if the Build User variable-extraction logic needs to change.